src/Stratis.Bitcoin.IntegrationTests/Mempool/MempoolRelaySpecification_Steps.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Linq;
+using FluentAssertions;
+using FluentAssertions.Common;
+using NBitcoin;
+using Stratis.Bitcoin.Connection;
+using Stratis.Bitcoin.Features.Consensus;
+using Stratis.Bitcoin.IntegrationTests.EnvironmentMockUpHelpers;
+using Xunit.Abstractions;
+
+namespace Stratis.Bitcoin.IntegrationTests.Mempool
+{
+    public partial class MempoolRelaySpecification
+    {
+        private NodeBuilder nodeBuilder;
+        private CoreNode nodeA;
+        private CoreNode nodeB;
+        private CoreNode nodeC;
+        private int coinbaseMaturity;
+        private Transaction transaction;
+
+        // NOTE: This constructor is allows test steps names to be logged
+        public MempoolRelaySpecification(ITestOutputHelper outputHelper) : base(outputHelper)
+        {
+        }
+
+        protected override void BeforeTest()
+        {
+            this.nodeBuilder = NodeBuilder.Create();
+        }
+
+        protected override void AfterTest()
+        {
+            this.nodeBuilder.Dispose();
+        }
+
+        protected void nodeA_nodeB_and_nodeC()
+        {
+            this.nodeA = this.nodeBuilder.CreateStratisPowNode();
+            this.nodeB = this.nodeBuilder.CreateStratisPowNode();
+            this.nodeC = this.nodeBuilder.CreateStratisPowNode();
+
+            this.nodeBuilder.StartAll();
+            this.nodeA.NotInIBD();
+            this.nodeB.NotInIBD();
+            this.nodeC.NotInIBD();
+
+            this.coinbaseMaturity = (int)this.nodeA.FullNode.Network.Consensus.Option<PowConsensusOptions>().CoinbaseMaturity;
+
+            // add some coins to nodeA
```

Comment:
this block deserves its own line as a `Given`

Reply:
yeah perhaps, it makes the test more reasable